tests/snippets/bytearray.py

```diff
@@ -29,3 +29,7 @@
 
 assert bytearray(b'Is Title Case').istitle()
 assert not bytearray(b'is Not title casE').istitle()
+
+a = b'abcd'
+a.clear()
```

Comment:
In cpython, `bytes` objects do not have a `clear` method. 
```
>>> a = b'abcd'
>>> a.clear()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'bytes' object has no attribute 'clear'
```

This makes sense, because byte strings are immutable. Did you mean to add this to `bytearray`?

```
>>> a = bytearray(b'abcd')
>>> a.clear()
>>> assert len(a) == 0
```

Reply:
Yes, I thought about `bytearray.clear`. Should I move it to `objbytearray.rs`?

Reply:
Yep!

Reply:
Fixed